api/swagger.yml

```
@@ -4259,6 +4284,13 @@ paths:
           type: string
           enum: [two_dot, three_dot]
           default: three_dot
+      - in: query
+        name: include_right
```

Consider
```suggestion
        name: include_right_info
```
Otherwise it is very confusing.